cpp/build.sh

```diff
@@ -115,58 +93,22 @@ function install_libtorch() {
   cd "$BWD" || exit
 }
 
-function install_yaml_cpp() {
-  YAML_CPP_SRC_DIR=$BASE_DIR/third-party/yaml-cpp
-  YAML_CPP_BUILD_DIR=$DEPS_DIR/yaml-cpp-build
-
-  if [ ! -d "$YAML_CPP_BUILD_DIR" ] ; then
-    echo -e "${COLOR_GREEN}[ INFO ] Building yaml-cpp ${COLOR_OFF}"
-
-    if [ "$PLATFORM" = "Linux" ]; then
-      SUDO="sudo"
-    elif [ "$PLATFORM" = "Mac" ]; then
-      SUDO=""
-    fi
-
-    mkdir $YAML_CPP_BUILD_DIR
-    cd $YAML_CPP_BUILD_DIR
-    cmake $YAML_CPP_SRC_DIR -DYAML_BUILD_SHARED_LIBS=ON -DYAML_CPP_BUILD_TESTS=OFF -DCMAKE_CXX_FLAGS="-fPIC"
```

Comment:
We need build yaml-cpp with DCMAKE_CXX_FLAGS="-fPIC"

Reply:
See above